src/univariate/discrete/binomial.jl

```diff
@@ -147,6 +147,10 @@ function mgf(d::Binomial, t::Real)
     n, p = params(d)
     (one(p) - p + p * exp(t)) ^ n
 end
+function cgf(d::Binomial, t)
+    n,p = params(d)
+    n*cgf(Bernoulli(p),t)
```

Comment:
To avoid checks or promotions of the argument:
```suggestion
    n * cgf(Bernoulli{typeof(p)}(p), t)
```